src/dropdown.ts

```diff
@@ -523,18 +531,18 @@ export default class Dropdown extends LitElement implements FormControl {
     // to select options based on the initial `value` and that the initial `value` is
     // the intended one. So we proceed.
     if (hasNoSelectedOptions) {
-      for (const option of this.#optionElements) {
-        // If `value` is an empty string, every option is left unselected. Otherwise,
-        // every `option.value` that's an empty string would be selected.
```

Comment:
No longer the case. The `find()` in the loop below prevents every option with an empty-string value from being selected. Also see the comment below that's above `!option.selected`.